packages/tpetra/core/src/Tpetra_CrsGraph_decl.hpp

```diff
@@ -1494,19 +1494,33 @@ namespace Tpetra {
     setAllIndices (const Teuchos::ArrayRCP<size_t> & rowPointers,
                    const Teuchos::ArrayRCP<local_ordinal_type> & columnIndices);
 
-    /// \brief Get a host view of the row offsets.
-    ///
-    /// \note Please prefer getLocalGraph() to get the row offsets.
+    /// \brief Get a host view of the packed row offsets.
     ///
+#ifdef TPETRA_ENABLE_DEPRECATED_CODE
     /// This may return either a copy or a view of the row offsets.
     /// In either case, it will <i>always</i> live in host memory,
     /// never in (CUDA) device memory.
     Teuchos::ArrayRCP<const size_t> getNodeRowPtrs () const;
```

Comment:
does this need a TPETRA_DEPRECATED marking?

Reply:
Good catch.  Added.